dashboards/google-backupdr/backupdr-monitoring.json

```
@@ -0,0 +1,157 @@
+{
+  "displayName": "Backup and DR Monitoring",
```

Let's have this match the display name in the yaml file.
```suggestion
  "displayName": "Backup/Recovery storage pool utilization",
```

---

Done. Updated yaml file with the same display name